src/PIL/EpsImagePlugin.py

```diff
@@ -193,7 +193,11 @@ def _open(self) -> None:
         self.fp.seek(offset)
 
         self._mode = "RGB"
-        self._size = None
+
+        # The last occurrence of one of these values
+        # takes precedence over prior occurrences.
+        bounding_box = None
+        imagedata_size = None
```

Comment:
I'm not convinced that this is always the case.

https://web.mit.edu/PostScript/Adobe/Documents/5001.DSC_Spec.pdf
>  In the case of multiple header comments, the first comment encountered is
considered to be the truth. In the case of multiple trailer comments (those
comments that were deferred using the (atend) convention), the last
comment encountered is considered to be the truth.